handbook/engineering/web/goals.md

```diff
@@ -4,29 +4,24 @@
 
 ### Long term
 
-**_Deliver the full, unique value of [extensions](https://docs.sourcegraph.com/extensions) to our users._**
+**_The web team has three areas of focus: creating and maintaining a highly usable and intentionally designed webapp interface; delivering the full, unique value of [extensions](https://docs.sourcegraph.com/extensions) to our users; and developing code insights into an entirely new featureset._**
```

Comment:
Thought: Having three focuses is kind of a contradictory statement. Can one be focused on three things at once? This is not necessarily a problem with the wording, but may just be revealing something inherent to these goals. The goal of our previous goal was to have a single focus that other concerns can be folded into as horizontal concerns. We're giving that up, and that may be fine if we can actually deliver on all these things.

Reply:
I agree that we're giving that up, but I think it's too constricting to try and limit the web team goals to a single overarching narrative that's still specific enough to be meaningful. ("Make a user enjoy using and extending sourcegraph" encompasses all these things but could apply to many teams and tells you nothing – as soon as you get specific enough, it becomes a list of multiple goals.) Good to call this out, but I think it's a fact of our size and responsibility as a single team that we have multiple goals at the moment.